### PR TITLE
add event counts to average report

### PR DESCRIPTION
**Pull request by agramfort**

### CODE REVIEW

File: scripts/report/01-make_reports.py [hunk 1]
```diff
@@ -450,6 +451,13 @@ def run_report_average(session):
     evokeds = mne.read_evokeds(evoked_fname)
     fs_subjects_dir = config.get_fs_subjects_dir()
 
+    df_events = count_events(config.bids_root)
+    rep.add_htmls_to_section(
+        f'<center>{df_events.to_html()}</center>',
+        captions='Events counts',
```
Comment (hoechenberger) [inline scripts/report/01-make_reports.py:457]:
```suggestion
        captions='Event counts',
```

File: scripts/report/01-make_reports.py [hunk 2]
```diff
@@ -450,6 +451,13 @@ def run_report_average(session):
     evokeds = mne.read_evokeds(evoked_fname)
     fs_subjects_dir = config.get_fs_subjects_dir()
 
+    df_events = count_events(config.bids_root)
+    rep.add_htmls_to_section(
+        f'<center>{df_events.to_html()}</center>',
```
Comment (hoechenberger) [inline scripts/report/01-make_reports.py:456]: Let's make it easier to style this via CSS later
```suggestion
        f'<div class="event-counts">\n    {df_events.to_html()}\n</div>',
```

File: scripts/report/01-make_reports.py [hunk 3]
```diff
@@ -450,6 +451,15 @@ def run_report_average(session):
     evokeds = mne.read_evokeds(evoked_fname)
     fs_subjects_dir = config.get_fs_subjects_dir()
 
+    df_events = count_events(config.bids_root)
+    rep.add_htmls_to_section(
+        f'<div class="event-counts">\n'
```
Comment (agramfort) [inline scripts/report/01-make_reports.py:456]: but now it's not centered anymore @hoechenberger :(

Comment (hoechenberger) [reply scripts/report/01-make_reports.py:456]: How can we add custom CSS to the report?

**Comment by agramfort** [comment]
You need to edit the tempita template file

**Comment by hoechenberger** [comment]
Yes, I just wondered if we could add custom CSS programmatically when creating the report. It seems not, so I will have to change things upstream...

**Comment by agramfort** [comment]
I fear that’s the price to pay until the report is refactored

**Comment by hoechenberger** [comment]
@agramfort https://github.com/mne-tools/mne-python/pull/8762 should allow us to easily add custom CSS

**Comment by hoechenberger** [comment]
CSS we should add:

```css
.event-counts {
    display: -webkit-box;
    display: -ms-flexbox;
    display: -webkit-flex;
    display: flex;
    justify-content: center;
    text-align: center;
}
```

**Comment by agramfort** [comment]
ok!

>

**Comment by hoechenberger** [comment]
Added custom CSS; blocker is now https://github.com/mne-tools/mne-python/pull/8762

**Comment by hoechenberger** [comment]
@agramfort 
This is currently blocked by https://github.com/mne-tools/mne-bids/pull/682

**Comment by agramfort** [comment]
thx @hoechenberger !

**Pull request merged**